client/my-sites/stats/site.jsx

```diff
@@ -267,23 +265,75 @@ class StatsSite extends Component {
 					</div>
 				</div>
 				<JetpackColophon />
+			</>
+		);
+	}
+
+	renderEnableStatsModule() {
+		const { slug, siteId, path } = this.props;
+		return (
+			<EmptyContent
+				illustration="/calypso/images/illustrations/illustration-404.svg"
+				title={ translate( 'Looking for Stats?' ) }
+				line={ translate(
+					'Enable Site Stats below or visit {{a}}Traffic > Site Stats{{/a}} for more options',
```

Comment:
Since "Traffic" is now under "Marketing", let's make sure to prepend this path with `Marketing > `.